src/app/pages/psbt-request/components/psbt-displayer.tsx

```diff
@@ -0,0 +1,65 @@
+import { Box, Text, color } from '@stacks/ui';
+
+import { isUndefined } from '@shared/utils';
+
+import { isTypedArray } from '@app/common/utils';
+
+function wrapText(text: string): JSX.Element {
+  return <Text wordWrap="break-word">{text}</Text>;
+}
+
+function Value({ val }: { val: any }) {
```

Comment:
fn name

Reply:
Coming back to this bc was in-progress. I added comments for a v2 bc we are just delivering the hex in the UI for v1.